tdex/docker-compose.yml

```diff
@@ -0,0 +1,56 @@
+version: "3.8"
+
+services:
+  app_proxy:
+    environment:
+      APP_HOST: $APP_TDEX_CONNECT_IP
+      APP_PORT: $APP_TDEX_CONNECT_PORT
+  
+  tdexd:
+    container_name: "tdexd"
+    image: docker pull ghcr.io/tdex-network/tdexd:v0.8.15@sha256:e79aff49e5220d55df7c94d01e404e11b7e5cf094b459b3e6fbb918a7f2a40d6
```

Comment:
This looks like a typo. This should be: `ghcr.io/tdex-network/tdexd:v0.8.15@sha256:e79aff49e5220d55df7c94d01e404e11b7e5cf094b459b3e6fbb918a7f2a40d6`